google/cloud/documentai_toolbox/wrappers/entity.py

```diff
@@ -26,44 +27,73 @@
 
 @dataclasses.dataclass
 class Entity:
-    r"""Represents a wrapped documentai.Document.Entity.
+    """Represents a wrapped `documentai.Document.Entity`.
 
     Attributes:
+        shard_index (int):
+            Required. The index of the `google.cloud.documentai.Document` containing
+            the `google.cloud.documentai.Document.Entity`.
         documentai_entity (google.cloud.documentai.Document.Entity):
-            Required. The original google.cloud.documentai.Document.Entity object.
+            Required. The original `google.cloud.documentai.Document.Entity` object.
+        id (int):
+            Required. The unique identifier of the `Entity` in the `Document`.
         type_ (str):
             Required. Entity type from a schema e.g. "Address".
         mention_text (str):
             Optional. Text value in the document e.g. "1600 Amphitheatre Pkwy".
             If the entity is not present in
             the document, this field will be empty.
+        normalized_text (str):
+            Optional. Normalized text value in the document e.g. "1970-01-01".
+            If the entity is not present in
+            the document, this field will be empty.
+        start_page (int):
+            Required. `Page` containing the `Entity` or the first page of the
+            classification (for Splitter/Classifier processors).
+        end_page (int):
+            Required. Last page of the classification
+            (for Splitter/Classifier processors). Default is `0` for other processors.
+        normalized_vertices (List[documentai.NormalizedVertex]):
+            Optional. Vertices for bounding box of `Entity`.
     """
-    documentai_entity: documentai.Document.Entity = dataclasses.field(repr=False)
+
+    shard_index: int = dataclasses.field()
+    documentai_entity: dataclasses.InitVar[documentai.Document.Entity]
```

Comment:
Just to confirm, are we changing all the existing wrapped documentai messages to `InitVar`, and extract information from the those messages in `__post_init__`, but never keeping the whole documentai message as a field?

(Indeed, we should keep this consistent across all wrapper classes - although technically they are no longer "wrappers" in the usual sense.)

Reply:
That is true. I am changing the documentai messages to InitVar for all wrappers except the `Document` wrapper. The `documentai_document` contains all of the information used to populate the other wrappers.